### ISSUE TITLE
.coffeelintignore not being respected

### ISSUE THREAD

**Issue by mlarraz** [zmbush/coffeelint-ruby#34]
Support for a `.coffeelintignore` file was added to coffeelint back in [v1.6.0](https://github.com/clutchski/coffeelint/commit/91dc8bf4b076dd9f9702935b447053a39b274ae3), but apparently this never gets picked up when running `coffeelint.rb`.

**Comment by zmbush** [comment]
Yes, that change requires porting the node.js code to ruby. I don't have much time to work in this gem lately, but you are welcome to implement the feature yourself.

**Comment by mlarraz** [comment]
Gave it a shot on https://github.com/zmbush/coffeelint-ruby/pull/36. Haven't actually tested it yet, though
